ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -55,6 +55,26 @@ resources:
       cpu: "1"
       memory: 4Gi
 tests:
+- as: alibaba-ipi-fips-p2-f7
```

Comment:
For 4.12 and above, suggest to add another one, i.e. "alibaba-ipi-private" of the workflow "cucushift-installer-rehearse-alibabacloud-ipi-private", because in my mind CEE guy ever mentioned one customer was interested on IPI private on alibabacloud.

Reply:
done